server/protocol.c

```diff
@@ -1465,6 +1466,18 @@ static void apply_server_config(request_rec *r)
     r->per_dir_config = r->server->lookup_defaults;
 }
 
+AP_DECLARE(int) ap_assign_request(request_rec *r,
+                                  const char *method, const char *uri,
+                                  const char *protocol)
+{
+    /* dummy, for now */
+    (void)r;
+    (void)method;
+    (void)uri;
+    (void)protocol;
+    return 0;
+}
+
```

Comment:
The proper implementation will come in a later PR when we start using REQUEST buckets.